gridstatus/spp.py

```diff
@@ -193,6 +201,189 @@ def get_load_forecast(self, date, forecast_type="MID_TERM", verbose=False):
 
         return current_day_forecast
 
+    def get_solar_and_wind_load_forecast(self, date, end=None, verbose=False):
+        """Solar and wind forecast for upcoming days. In 5-minute intervals if end
+        is less than 4 hours from now. Otherwise in hourly intervals.
+
+        Returns:
+            pd.DataFrame: forecast for current day
+        """
+        now = pd.Timestamp.now(tz=self.default_timezone)
+
+        use_shortterm = False
+
+        # Only get the shortterm forecast if end is provided and end is less
+        # than 4 hours from now
+        if end:
+            end = utils._handle_date(end, self.default_timezone)
+            use_shortterm = end < now + pd.Timedelta(hours=4)
+            # Files don't exist in the future, so we need to limit the end.
+            end = min(end, now)
+
+        if date == "latest":
+            date = now
+
+        date = utils._handle_date(date, self.default_timezone)
+
+        if use_shortterm:
```

Comment:
remove all this extra logic if you can. these functions should focus on just scraping the data and light cleaning to column names. 

i dont see why this needs much more logic then 

`get_operating_reserves` or `get_day_ahead_operating_reserve_prices`

which also scrap 5 minute and hourly files